### PR TITLE
Add Trivy Scan and Structure Test

### LINKED ISSUE #67: Add trivy scan to CI
A trivy scan would be a nice addition for the Pipeline

https://github.com/aquasecurity/trivy

### PR DESCRIPTION
**Pull request by martialblog**
Ok, so here's the Trivy Scan and Structure Test. Without caching for now, cause it's a bit tricky, but I tried to write it as parallel as possible. 

Might need some work, but the structure is there. 

I'm not sure I fully understand the GitHub Code-Ccanning Tab yet...

Fixes #67

### CODE REVIEW

**Comment by martialblog** [comment]
See also #69

File: .github/workflows/test-lts-container-images.yaml [hunk 1]
```diff
@@ -0,0 +1,50 @@
+name: Test LTS Container Images
+
+# TODO: When to Trigger?
+on: [push]
```
Comment (jacksgt) [inline .github/workflows/test-lts-container-images.yaml:4]: Looking at the available options, I think this makes the most sense.
https://docs.github.com/en/actions/reference/events-that-trigger-workflows

**Review by jacksgt — APPROVED** [review]
Excellent work Markus! Shipit.

File: .github/workflows/test-lts-container-images.yaml (hunk 1 above)
Comment (martialblog) [reply .github/workflows/test-lts-container-images.yaml:4]: :+1:  I might add pull_request

**Pull request merged**